### PR TITLE
Remove unused Grafana dashboards

### PR DESCRIPTION
**Pull request by rjbaker**
These dashboards are deployed with Grafana and their configuration is managed in this repo.

We have copied some these dashboards to an editable version, allowing editors to make changes within Grafana. Our Grafana instance is backed by a PostgreSQL service allowing editors to update dashboards and persist changes directly.

Some dashboards are removed as they are not relevant to the Notify service.

### CODE REVIEW

**Review by sakisv — APPROVED** [review]
Screenshot 2022-06-15 at 16 17 15

👀 💯

**Comment by idavidmcdonald** [comment]
Nice, what are your thoughts on getting rid of the rest of the dashboards from that folder?

Potentially if we want to keep them we could instead move them into the Notify folder instead (although I think one may have already been moved?).

File: README.md [hunk 1]
```diff
@@ -197,3 +173,27 @@ docker_credentials = {
...
+```
+terraform init
+gds aws notify-tools-admin -- terraform apply
+```
+
+You will be prompted to enter your GOV.UK PaaS authentication credentials or and SSO passcode if SSO is enabled.
+
+Obtain an SSO passcode by signing into GOV.UK PaaS and navigating to: [login.cloud.service.gov.uk/passcode](https://login.cloud.service.gov.uk/passcode)
+
+Copy the passcode and paste into the terminal prompt.
```
Comment (sakisv) [inline README.md]: It's worth pointing out that this is something that they'll need to do only once per day (I think PaaS tokens have a TTL of 8h?)

File: README.md [hunk 2]
```diff
@@ -197,3 +173,27 @@ docker_credentials = {
...
 }
+```
+
+## Terraform Deployment
+
+The deployment of the `prometheus_all` module is managed in the [deployment](deployment) directory. This includes all internal and external metric target configuration.
+
+### Applying Terraform
+
+Terraform is applied to the `monitoring` space of the GOV.UK Notify Org - you must have admin access to this space.
```
Comment (sakisv) [inline README.md]: I think `space developer` is enough

File: README.md (hunk 1 above)
Comment (rjbaker) [reply README.md]: Mine asks every time for some reason

**Comment by rjbaker** [comment]
> Nice, what are your thoughts on getting rid of the rest of the dashboards from that folder?
> 
> Potentially if we want to keep them we could instead move them into the Notify folder instead (although I think one may have already been moved?).

We've copied the InfluxDB dashboard so we can make changes easily. I've also copied the "Advanced PostgreSQL" board as it shows some more advanced Postgres metrics than Notify's homemade database dashboards. I've then updated the config so it's relevant to Notify - but it's essentially a copy of this one: https://grafana.com/grafana/dashboards/9628

I'm inclined the keep the others where they are as they're immutable and provide an excellent overview of what's happening across all spaces. Also, in the event we lost Grafana's Postgres data, until recovery, we would still have some basic dashboards.

File: README.md (hunk 1 above)
Comment (sakisv) [reply README.md]: It asks, but you can just hit enter without passing anything in and it will work

**Pull request merged**